.github/workflows/website.yml

```diff
@@ -3,8 +3,8 @@ name: Render Website
 on:
   push:
     branches:
-      - 'main'
```

Comment:
My formatter went through and updated all of these. If you would prefer single quotes I can go back and revert those changes.